publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandler.java

```diff
@@ -0,0 +1,166 @@
+    For a full list of individual contributors, please see the commit history.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+package com.ericsson.eiffel.remrem.publish.service;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.spi.json.JacksonJsonNodeJsonProvider;
+import com.jayway.jsonpath.spi.mapper.JacksonMappingProvider;
+import org.apache.commons.io.IOUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class EventTemplateHandler {
+    private final Configuration configuration = Configuration.builder()
+            .jsonProvider(new JacksonJsonNodeJsonProvider())
+            .mappingProvider(new JacksonMappingProvider())
+            .build();
+
+    // Paths in Semantics JAR
+    private final String EVENT_TEMPLATE_PATH = "templates/";
+    private final String EVENT_SCHEMA_PATH = "schemas/input/";
+
+    //private JsonNode updatedJson;
+
+    // eventTemplateParser
+    public JsonNode eventTemplateParser(String json_data , String event_name){
+        JsonNode updatedJson = null;
+        JsonFactory factory = new JsonFactory();
+        ObjectMapper mapper = new ObjectMapper(factory);
+        JsonNode rootNode = null;
+        try {
+            String event_template = accessFileInSemanticJar(EVENT_TEMPLATE_PATH + event_name.toLowerCase() + ".json");
+
+            rootNode = mapper.readTree(json_data);
+            updatedJson = mapper.readValue(event_template, JsonNode.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        // For each key/value pair for parsing to template
+        Iterator<Map.Entry<String,JsonNode>> fieldsIterator = rootNode.fields();
+        while (fieldsIterator.hasNext()) {
+            Map.Entry<String, JsonNode> field = fieldsIterator.next();
+            // Parse values to template
+            // Check if POJO required for update in event template
+            Pattern p = Pattern.compile("\\[\\d+\\]$");  // if ends with [d+]
```

Comment:
create private static final SMTH = "\\[\\d+\\]$" and use this variable everywhere instead of dublicate this string